api/app.py

```diff
@@ -112,6 +113,33 @@ def handle_delete_user():
     return Response(body=None, status_code=204)
 
 
+@app.route("/user/manage_preferences", methods=["POST"])
+def manage_preferences():
+    """Checks for user from email property in request body
+    and sends an email to containing a link to update notification
+    preferences with a temp auth token included
+
+    Returns:
+        [Response]: A formatted response to the API caller
+    """
+    user_email = app.current_request.json_body.get('email')
+    if user_email:
+        try:
+            user = fetch_user(user_email)
+            send_emails_to_users([user])
```

Comment:
This should be `send_emails_to_users([user], EmailTemplate. UpdatePreferences)`

Since you can't test these changes out though so probably my fault 😄